actors/builtin/storage_power/policy.go

```diff
@@ -34,16 +35,18 @@ func pledgePenaltyForConsensusFault(currPledge abi.TokenAmount, faultType Consen
 }
 
 func rewardForConsensusSlashReport(elapsedEpoch abi.ChainEpoch, collateralToSlash abi.TokenAmount) abi.TokenAmount {
-	autil.TODO()
-	// BigInt Operation
-	// var growthRate = builtin.SLASHER_SHARE_GROWTH_RATE_NUM / builtin.SLASHER_SHARE_GROWTH_RATE_DENOM
-	// var multiplier = growthRate^elapsedEpoch
-	// var slasherProportion = min(INITIAL_SLASHER_SHARE * multiplier, 1.0)
-	// return collateralToSlash * slasherProportion
-	return abi.NewTokenAmount(0)
+	// autil.PARAM_FINISH()
+	growthRate := big.Div(indices.ConsensusFault_SlasherShareGrowthRateNum(), indices.ConsensusFault_SlasherShareGrowthRateDenom())
```

Comment:
This evaluates to a constant `1`. The point of separating numerator and denominator is to re-arrange the arithmetic to do division last. Please document the canonical expression in a comment, and then code the arithmetic to work with integers.